config/deploy/staging.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-set :branch, :test
+set :branch, :main
```

Comment:
I think that having this different should signify that changes are being tested on the test then merged into main. Maybe that hasn't often happened, but to me it feels more correct to leave this at test.

From your comments @Br3nda it sounds like you're used to something different though. I don't think it matters a huge amount anyway as this can be set manually any time someone is deploying if they want to use a different branch.

I'm going to go ahead and approve this - easy to come back and tweak later if a need arises.